corehq/apps/users/models_sql.py

```diff
@@ -179,6 +184,41 @@ class RoleAssignableBy(models.Model):
     )
 
 
+class UserUpdateMeta(jsonobject.JsonObject):
+    updated_via = jsonobject.StringProperty()
+
+
+class UpdateDetails(jsonobject.JsonObject):
+    meta = jsonobject.ObjectProperty(UserUpdateMeta)
+    changes = jsonobject.DictProperty()
+
+
+class HQLogEntry(models.Model):
+    """
+    HQ Adaptation of Django's LogEntry model
+    """
+    CREATE = 1
+    UPDATE = 2
+    DELETE = 3
+
+    ACTION_FLAG_CHOICES = (
+        (CREATE, _('Create')),
+        (UPDATE, _('Update')),
+        (DELETE, _('Delete')),
+    )
+    domain = models.CharField(max_length=255, db_index=True)
```

Comment:
indexes should be added via the class `Meta` to avoid creating search indexes which aren't needed